documentation/open-api/swagger.yaml

```diff
@@ -17,6 +17,10 @@ tags:
   description: Everything about invigilators
 - name: Rooms
   description: Everything about rooms
+- name: Appointments
+  description: Everything about appointments
+- name: ExamTypes
```

Comment:
Should `ExamTypes` be part of exams?

Reply:
![Screen Shot 2019-03-29 at 9 31 51 AM](https://user-images.githubusercontent.com/29988052/55247721-93d47800-5205-11e9-9e7e-b826f4871616.png)

Reply:
Did it based on the folder structure we have in resources.

Reply:
Also, exam_types is its own model, just my reasoning behind it.

Reply:
Weird. I wonder why we did it that way. This makes sense based on that I guess